docs/mpcie_modules.md

```diff
@@ -162,6 +162,50 @@ operating system.
 
 (1) Refer to quirks, it is known to not work stably on all apu2 board revisions
 
+## Ethernet controllers
+
+### Dual Ethernet Controller Realtek RTL8111
+
+![RTL8111](images/realtek_controller.jpg)
+
+**Compatible slots**
+
+| Platform |  mPCIe1  |     mPCIe2      |  mSATA   |
+|:--------:|:--------:|:---------------:|:--------:|
+|   apu2   | &#10004; | &#10004; &sup1; | &#10006; |
+|   apu3   | &#10004; | &#10006;        | &#10006; |
+|   apu4   | &#10004; | &#10006;        | &#10006; |
+
+(1) If RTL8111 is attached to mPCIe2 slot then option *force mPCIe2 slot Clk* in
+Payload menu must be enabled
+
+This model works well with apu1 boards. However, on apu2, apu3 and apu4
+platforms following steps should be done to correctly enable it:
+
+1. On apu2 board make sure to enable option *force mPCIe2 slot Clk* in Payload
+menu if you attached module to mPCIe2 slot. This menu is available during boot
+process by pressing `F10` button. In OS, you should be able to see your Ethernet
+controller under PCI devices.
```

Comment:
Note it is eligible only on apu2 @piotr-kleins

Reply:
I mentioned that other apus are not affected by that option

Reply:
@piotr-kleins my bad. I have overseen that.